include/envoy/http/access_log.h

```diff
@@ -164,7 +175,7 @@ class Formatter {
                              const RequestInfo& request_info) const PURE;
 };
 
-typedef std::unique_ptr<Formatter> FormatterPtr;
+typedef std::shared_ptr<Formatter> FormatterPtr;
```

Comment:
Also, is there a good reason this is now shared? Shared ownership semantics are more expensive and harder to reason about, so all things being equal it's best to keep unique. If there is a reason, please share it in some comment.